### DISCUSSION TITLE
SIGNUP_ENABLED variable not working for me.

### DISCUSSION THREAD

**Discussion by Entheologian** [Dan6erbond/share-me#18] [General]
Hi,

As per this info:
> If you only want to disable sign-up, remove the "create" API rules to only allow admins to create new users. You can set the SIGNUP_ENABLED environment variable in the frontend to disable the sign-up page and remove links to it.

I tried setting this as such in my docker-compose.yml file but it doesn't seem to work for me using this:
```
environment:
   - SIGNUP_ENABLED=false
```

The sign-up link is still present and if I also set the "create" API to admins only, the sign-ups still allow sign-ups.

Am I doing this incorrectly? I don't see any errors to go on.. everything so far works great.

**Comment by JPDVM2014** [comment]
Ever get this working? I'm having the same issue.
